src/main/java/de/chojo/repbot/ReputationBot.java

```diff
@@ -237,14 +238,27 @@ private void initLocalization() {
     }
 
     private void initBot() {
+        metrics = new Metrics(dataSource);
         RestAction.setDefaultFailure(throwable -> {
             if (throwable instanceof InsufficientPermissionException) {
                 PermissionErrorHandler.handle((InsufficientPermissionException) throwable, shardManager, localizer, configuration);
                 return;
             }
+            if (throwable instanceof ErrorResponseException e) {
+                if (e.getErrorResponse() == ErrorResponse.UNKNOWN_INTERACTION) {
+                    metrics.service().failedInteraction();
```

Comment:
This will work in this case as I think we do not have any custom error handlers when acknowledging an interaction. However, this will not capture failed interactions when we have an own failure handler in the queue() call.

Reply:
Actually no.
We use the ErrorResponseHandler#ignore call nearly everywhere. This ignores matching exceptions or passes them into the default failure if the exception does not match.
![image](https://user-images.githubusercontent.com/46890129/179950649-b30342fc-0f16-4387-a75b-bfcbc935f564.png)